Kickstarter-iOS/Views/Cells/PledgeShippingLocationCell.swift

```diff
@@ -86,8 +94,6 @@ final class PledgeShippingLocationCell: UITableViewCell, ValueCell {
 private let amountLabelStyle: LabelStyle = { (label: UILabel) in
   label
     |> \.adjustsFontForContentSizeCategory .~ true
-    |> \.font .~ UIFont.ksr_title1()
```

Comment:
This is no longer needed...these are instead passed to the formatter/attributter